pkg/querier/tsdb_ingester_streaming_queryable.go

```diff
@@ -0,0 +1,66 @@
+package querier
+
+import (
+	"unsafe"
+
+	"github.com/cortexproject/cortex/pkg/ingester/client"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/storage"
+)
+
+// TimeSeriesSeriesSet is a wrapper around a client.TimeSeries slice to implement to SeriesSet interface
+type TimeSeriesSeriesSet struct {
```

Comment:
Because it is a wrapper around a `TimeSeries` to implement the `SeriesSet` interface. So I thought that name although a little ugly better indicated what it was.

Reply:
Ok. Thanks for the explaining the rationale.